chromadb/api/models/Collection.py

```diff
@@ -44,15 +46,14 @@
     validate_embeddings,
     validate_embedding_function,
 )
-import logging
 
 logger = logging.getLogger(__name__)
 
 if TYPE_CHECKING:
     from chromadb.api import ServerAPI
 
 
-class Collection(BaseModel):
+class Collection(BaseModel, Configurable):
```

Comment:
I think the semantic trick here is to not use this as the model, its really just an API wrapper. What I did in #1491 is to remove this clunkiness altogether and make the colleciton a wrapper around the _actual_ model. This was tech debt we needed to pay.